pkg/storage/etcd3/store_test.go

```diff
@@ -22,17 +22,18 @@ import (
 	"sync"
 	"testing"
 
+	apitesting "k8s.io/apimachinery/pkg/api/testing"
```

Comment:
> it's where the fuzzer went. Am undecided yet whether it's good or bad: it's where I create a scheme, codecs, etc. for tests, registering meta types and client core types. The later should be replaced with real test group types in a follow-up.

I don't want a global testing scheme, codec, registry.  I think that worked out badly.  Helpers to make it easy to create a local one are fine and re-using a real one (like from client-go) is also fine.